frontend/public/components/routes/create-route.tsx

```diff
@@ -196,11 +227,54 @@ export class CreateRoute extends React.Component<null, CreateRouteState> {
       }));
   }
 
+  newAltServiceEntry() {
+    return {
+      entry: {
+        service: null,
+        weight: 100,
+      },
+      uid: _.uniqueId(),
```

Comment:
I'd call this `key` since that's how it's used, and it avoids the impression that this is `service.metadata.uid`

Reply:
I'd think I'd just flatten this structure and remove the `uid` (or `key`) props when submitting. I'd also use the prop `name` to match the `alternateBackends` object

```js
return {
  name: null,
  weight: 100,
  key: _.uniqueId('alternate-backend-'),
};
```

Reply:
You should define a type for this to take advantage of type checking